CMakeLists.txt

```diff
@@ -9,7 +9,16 @@ project(target-isns "C")
 set(TARGET_ISNS_VERSION "0.6.1")
 
 cmake_minimum_required(VERSION 2.8)
+
+set(IS_CMAKE_31 false)
+if(NOT CMAKE_VERSION VERSION_LESS "3.1")
+  set(IS_CMAKE_31 true)
```

Comment:
Please drop the `IS_CMAKE_31` variable since we will remove some locations it will be needed.